index.html

```diff
@@ -303,85 +300,31 @@
-        contains a list of <a>PerformanceEntry</a> objects, sorted in
-        chronological order with respect to <a for=
-        "PerformanceEntry">startTime</a>, that match the
-        following criteria:
-      </p>
-      <ol>
-        <li>Let the <dfn>list of entry objects</dfn> be the empty
-        <a>PerformanceEntryList</a>.
-        </li>
-        <li>Let the <dfn>set of filter properties</dfn> be a set of pairs
-        where the first element is the _name_ of a [dictionary member] of
-        `filter` that is present and the second element is
-        the _value_ of that [dictionary member].
-        </li>
-        <li>For each <a>PerformanceEntry</a> object
-        (<dfn>entryObject</dfn>) in the <a>performance entry buffer</a>, in
-        chronological order with respect to <a for=
-        "PerformanceEntry">startTime</a>:
-        <ol>
-          <li>For each _name_ and _value_ pair in <a>set of filter
-          properties</a>:
-            <ol>
-              <li>If the <a>entryObject</a> does not contain an attribute
-              whose name matches _name_ in a [case-sensitive] manner, go to next <a>entryObject</a>.
-              </li>
-              <li>Otherwise, if the <a>entryObject</a> contains an
-              attribute whose name matches _name_ in a [case-sensitive] manner, and its value does not
-              match _value_ in a [case-sensitive] manner, go to next <a>entryObject</a>.
-              </li>
-            </ol>
-          </li>
-          <li>Add <a>entryObject</a> to the <a>list of entry objects</a>.
-            </li>
-          </ol>
-        </li>
-        <li>Return the <a>list of entry objects</a>.
-        </li>
-      </ol>
+        method returns a <a>PerformanceEntryList</a> object returned by
+        <a href="#filter-performance-entry-buffer-by-name-and-type"></a>
+        algorithm with <var>name</var> and <var>type</var> set to `null`.</p>
       <p>
         The <dfn for="Performance">getEntriesByType</dfn> method
         returns a <a>PerformanceEntryList</a> object returned by
-          <a for='Performance' data-lt='getEntries'>getEntries({'entryType': type})</a>.
+        <a href="#filter-performance-entry-buffer-by-name-and-type"></a>
+        algorithm with <var>name</var> set to `null` and <var>type</var> set to
+        `type`.
       </p>
       <p>
         The <dfn for="Performance">getEntriesByName</dfn> method
         returns a <a>PerformanceEntryList</a> object returned by
-        <a for='Performance' data-lt='getEntries'>getEntries({'name': name})</a>
-        if optional `entryType` is omitted, and
-        <a for='Performance' data-lt='getEntries'>getEntries({'name': name, 'entryType': type})</a>
-        otherwise.
+        <a href="#filter-performance-entry-buffer-by-name-and-type"></a>
+        algorithm with <var>name</var> set to `name` and <var>type</var> set to
+        `null` if optional `entryType` is omitted, and <var>type</var> set to
```

Comment:
there is no type argument for getEntriesByName. Should say:
 ... with <var>name</var> set to `name` and <var>type</var> set to  `null`.